packages/foam-vscode/jest.config.js

```diff
@@ -91,7 +91,7 @@ module.exports = {
   // notifyMode: "failure-change",
 
   // A preset that is used as a base for Jest's configuration
-  // preset: undefined,
+  preset: 'ts-jest',
```

Comment:
Both babel-jest and ts-jest were installed. Since ts-jest seems to be the more complete and powerful option, and also it's simpler to configure, I removed babel-jest. It's pointless to have both.

Reply:
Good call